lgc/interface/lgc/Builder.h

```diff
@@ -760,8 +760,10 @@ class Builder : public BuilderBase {
   // Create a buffer length query based on the specified descriptor.
   //
   // @param bufferDesc : The buffer descriptor to query.
+  // @param offset: The result is clamped at 0 for a null descriptor.
```

Comment:
Hmm, still not sure about the documentation. How about:
1. Change the top line to "Create a buffer length query based on the specified descriptor, subtracting an offset from the length. The result is 0 for a null descriptor when robustAccess is enabled.
2. Change the param line to "@param offset: The offset to subtract from the buffer length"

Reply:
It's pretty good. Done.